buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -618,7 +618,25 @@ def set_build_variables() {
 def set_sdk_variables() {
     DATESTAMP = get_date()
     SDK_FILE_EXT = SPEC.contains('zos') ? '.pax' : '.tar.gz'
-    SDK_FILENAME =  "OpenJ9-JDK${SDK_VERSION}-${SPEC}-${DATESTAMP}${SDK_FILE_EXT}"
+    // Get full version from tag (eg. 11.0.8+10) and replace + with _
+    FULL_SDK_VERSION = sh (
+        script: """grep OPENJDK_TAG closed/openjdk-tag.gmk | awk 'BEGIN {FS = "[ :=]+"} {gsub(/\\+/, "_")} {print \$2}'""",
```

Comment:
Command could be simplified by using `sed`:
 `sed -n -e 's/+/_/g; s/^OPENJDK_TAG := //p' < closed/openjdk-tag.gmk`

Reply:
I don't think either option is simple :laughing: 
I ~stole~ borrowed it from Adopt
https://github.com/AdoptOpenJDK/openjdk-build/blob/9148896926f269c42d36d40a9edd9248b0d601cb/sbin/build.sh#L728
Does it work on z/OS? I will reconfirm if the awk works on z/OS.

Reply:
Both ways (sed or awk) work on z/OS. Is the tag file guaranteed to only ever have a single line?

Reply:
Updated to sed. Also changed the filename def to be set based on BUILD_IDENTIFIER.